### ISSUE TITLE
Item Attributes not stored properly.

### ISSUE THREAD

**Issue by blablubbabcDEV** [Shopkeepers/Shopkeepers#409]
**Migrated from:** https://dev.bukkit.org/projects/shopkeepers/issues/408

**Originally posted by rikulaxion1993 (May 13, 2016):** 

What steps will reproduce the problem?
1. Add any item with attributes specific to one slot "When in off hand." and add it to a villager shop.
2. Restart Server.
3. Item will return with all slots for that attribute. "When in Off hand, main hand, chest, leg, head, feet.What is the expected output? What do you see instead?
Upon shutting down the server for restarts the expected items are thought to store the item attributes to the correct "When on" slot.
Instead the item loses the data pertained to which "When on" slot and when the server starts back up, it has all the "When on" slots.What version of the product are you using?
Shopkeepers 1.73Do you have an error log of what happened?
I do believe there is no error log (one could not be found) as the plugin believes it is functioning properly.Please provide any additional information below.
I do believe this may have just been an oversight of updating to the 1.9 item attribute system.

**Comment by blablubbabcDEV** [comment]
**Originally commented by blablubbabc (May 13, 2016):** 

Hm, I actually thought I fixed something related to this already, see this old ticket: http://dev.bukkit.org/bukkit-plugins/shopkeepers/tickets/367-1-9-attributes-are-not-properly-kept/
I will try to reproduce this on my test server, maybe I have missed something.
Update: Okay, I tried this on my test server and I cannot reproduce your issue. For me it does properly restore the attributes on reloads/server restarts.

Shopkeepers uses bukkit/spigot's item serialization API for attributes. So make sure that you are using the latest version of spigot, in case it is related to that.

            Edited May 14, 2016
